client/a8c-for-agencies/sections/marketplace/issue-license/hooks/use-assign-licenses-to-site.ts

```diff
@@ -25,7 +25,7 @@ const useAssignLicensesToSite = (
 	assignLicensesToSite: ( licenseKeys: string[] ) => Promise< PurchasedProductsInfo >;
 	isReady: boolean;
 } => {
-	const products = useProductsQuery( true );
+	const products = useProductsQuery();
```

Comment:
I haven't tested it yet, but I'm wondering if we are changing behavior here, or we want to use `usePublicProductsQuery`?

Reply:
We used to fetch the products using a public API earlier since we didn't have the actual API. Now that it is added, we are switching to that.

Reply:
For context, we initially used the public API to demonstrate a working Marketplace UI while we waited for the Billing infrastructure. This PR intends to update this and replace it with the non-public Jetpack licensing API, as it now works with A4A authentication. Also the public API is only limited to Jetpack Manage and would not load the correct products for A4A even with the authentication.

The data structure for the public and private APIs is the same, so no changes in the presentation layer should be required.